mdnsbridge/mdnsbridgeclient.py

```diff
@@ -46,11 +46,11 @@ def getHref(self, srv_type, priority=None, api_ver=None, api_proto=None, api_aut
                 # Re-try after cache has been updated
                 return self.getHrefWithException(srv_type, priority, api_ver, api_proto, api_auth)
         except NoService:
-            self.logger.writeWarning("No DNS-SD service for for {}, priority={}, api_ver={}, api_proto={}".format(
+            self.logger.writeWarning("No DNS-SD service for {}, priority={}, api_ver={}, api_proto={}".format(
```

Comment:
Sorry, only spotted this as a result of your typo fix. Could you add the value of api_auth to the logging here?